cloud_governance/common/clouds/aws/utils/utils.py

```diff
@@ -72,17 +72,18 @@ def tag_aws_resources(self, client_method: Callable, tags: list, resource_ids: l
         :param resource_ids:
         :return:
         """
-        proc = []
-        bulk_resource_ids_list = self.__split_run_bulks(iterable=resource_ids, limit=self.__update_tag_bulks)  # split the aws resource_ids into batches
-        co = 0
-        cpu_based_resource_ids_list = self.__split_run_bulks(iterable=bulk_resource_ids_list, limit=self.__update_tag_bulks)
-        for cpu_based_resource_ids_list in cpu_based_resource_ids_list:
-            for resource_ids_list in cpu_based_resource_ids_list:
-                p = Process(target=self.__tag_resources, args=(client_method, resource_ids_list, tags, ))
-                co += 1
-                p.start()
-                proc.append(p)
-            for p in proc:
-                p.join()
-        return co
+        if tags:
```

Comment:
I think that no need to call to tag_aws_resources if tags empty

Reply:
yes, This is the global place to check tags empty.